amxmodx/CLang.cpp

```diff
@@ -318,8 +318,9 @@ void reparse_color(String* def)
 // -- BAILOPAN
 int CLangMngr::MergeDefinitionFile(const char *file)
 {
-	FILE *fp = fopen(file, "rt");
-	if (!fp)
+	const char* md5buffer = hashFile((const char *)&file[0], Hash_Md5);
```

Comment:
Not really as I see. Let's keep things as they were, rolling back action, change &Var[0] to Var.